gridmenuview/src/main/java/com/bcc/gridmenuview/GridMenu.java

```diff
@@ -59,6 +63,17 @@ public void setOnClickListener(OnItemClickListener listener) {
         adapter.setOnClickListener(listener);
     }
 
+    private enum ShapeElement {
```

Comment:
you can separate ShapeElement to another file (class)

Reply:
thanks for the advice I'll improve the code